app/controllers/followings_controller.rb

```diff
@@ -0,0 +1,26 @@
+class FollowingsController < ApplicationController
+  before_action :require_user
+
+  def create
+    @following = current_user.followings.build(follow_id: params[:follow_id])
```

Comment:
Also, since you always redirect in this action, you don't need the variable to be an instance variable. When you redirect, you lose your instance variables and they have to be setup in the next controller action.